ingenialink/ethercat/network.py

```diff
@@ -76,6 +80,8 @@ class EthercatNetwork(Network):
     def __init__(
         self, interface_name: str, connection_timeout: float = DEFAULT_ECAT_CONNECTION_TIMEOUT
     ):
+        if not pysoem:
+            raise ImportError("DLLs required not found: Please install WinPcap and restart")
```

Comment:
This could be masking other import errors.

Something like this could be done:

```
try:
    import pysoem  # type: ignore
except ImportError as ex:
    pysoem = None
    pysoem_import_error = ex
```

```
if not pysoem:
    raise pysoem_import_error ?
```

Or saving the message instead of the exception object

Reply:
Mmm, I like it. With this change, we can not say explicitly that WinPcap is needed, but we will not mask any error. I prefer it.